nextmv/nextmv/cloud/application.py

```diff
@@ -37,24 +38,57 @@ class DownloadURL(BaseModel):
 
 
 class PollingOptions(BaseModel):
-    """Options to use when polling for a run result."""
-
-    backoff: float = 1
-    """Backoff factor to use between polls. Leave this at 1 to poll at a
-    constant rate."""
-    delay: float = 1
-    """Delay to use between polls, in seconds."""
+    """
+    Options to use when polling for a run result.
+
+    The Cloud API will be polled for the result. The polling stops if:
+
+    * The maximum number of polls (tries) are exhausted. This is specified by
+      the `max_tries` parameter.
+    * The maximum duration of the polling strategy is reached. This is
+      specified by the `max_duration` parameter.
+
+    Before conducting the first poll, the `initial_delay` is used to sleep.
+    After each poll, a sleep duration is calculated using the following
+    strategy, based on exponential backoff with jitter:
+
+    ```
+    sleep_duration = min(`max_delay`, `delay` + `backoff` * 2 ** i + Uniform(0, `jitter`))
+    ```
+
+    Where:
+    * i is the retry (poll) number.
+    * Uniform is the uniform distribution.
+
+    Note that the sleep duration is capped by the `max_delay` parameter.
+    """
+
+    backoff: float = 0.9
+    """
+    Exponential backoff factor, in seconds, to use between polls. Leave this at
+    1 to poll at a constant rate.
```

Comment:
```suggestion
    Exponential backoff factor, in seconds, to use between polls.
```

I think the comment was outdated and maybe not necessary? Just a recommendation though. Feel free to add more context. :blush: